test/e2e/upgradetest/resize_test.go

```diff
@@ -29,6 +29,9 @@ func TestResize(t *testing.T) {
 		if err != nil {
 			t.Fatal(err)
 		}
+		if err = e2eutil.WaitUntilDeploymentDeleted(testF.KubeCli, 3*time.Second, 30*time.Second, "etcd-operator", testF.KubeNS); err != nil {
```

Comment:
Yes, when I ran both my tests together one after another, the the test that ran second failed because the previous deployment was still being deleted:
```
--- FAIL: TestResize (0.13s)
	resize_test.go:35: object is being deleted: deployments.extensions "etcd-operator" already exists
FAIL
FAIL	github.com/coreos/etcd-operator/test/e2e/upgradetest	96.266s
```